### ISSUE TITLE
No responding after some days

### ISSUE THREAD

**Issue by vplustute** [nfarina/homebridge#2029]
Hello,

I installed Homebridge on a rasperrry Pi 3B and everything is working fine.
But after some days when I go in the Home App and at all items I get the Information no update.
Them I restart the Homebridge and everything I working fine again.

Here is you can see my system log file.

> Oct  7 06:47:01 raspberrypi CRON[16128]: (root) CMD (test -x /usr/sbin/anacron || ( cd / && run-parts --report /etc/cron.weekly ))
Oct  7 07:17:01 raspberrypi CRON[16758]: (root) CMD (   cd / && run-parts --report /etc/cron.hourly)
Oct  7 08:17:01 raspberrypi CRON[18010]: (root) CMD (   cd / && run-parts --report /etc/cron.hourly)
Oct  7 08:52:00 raspberrypi systemd[1]: Starting Daily apt download activities...
Oct  7 08:52:02 raspberrypi systemd[1]: Started Daily apt download activities.
Oct  7 08:52:02 raspberrypi systemd[1]: apt-daily.timer: Adding 11h 48min 22.279353s random time.
Oct  7 08:52:02 raspberrypi systemd[1]: apt-daily.timer: Adding 8h 3min 35.173561s random time.
Oct  7 09:17:01 raspberrypi CRON[19426]: (root) CMD (   cd / && run-parts --report /etc/cron.hourly)
Oct  7 10:17:01 raspberrypi CRON[20743]: (root) CMD (   cd / && run-parts --report /etc/cron.hourly)
Oct  7 11:17:01 raspberrypi CRON[22046]: (root) CMD (   cd / && run-parts --report /etc/cron.hourly)
Oct  7 12:17:01 raspberrypi CRON[23350]: (root) CMD (   cd / && run-parts --report /etc/cron.hourly)
Oct  7 13:17:01 raspberrypi CRON[24785]: (root) CMD (   cd / && run-parts --report /etc/cron.hourly)
Oct  7 14:17:01 raspberrypi CRON[26074]: (root) CMD (   cd / && run-parts --report /etc/cron.hourly)
Oct  7 15:17:01 raspberrypi CRON[27391]: (root) CMD (   cd / && run-parts --report /etc/cron.hourly)
Oct  7 16:17:01 raspberrypi CRON[28673]: (root) CMD (   cd / && run-parts --report /etc/cron.hourly)
Oct  7 16:19:31 raspberrypi systemd[1]: Starting Cleanup of Temporary Directories...
Oct  7 16:19:31 raspberrypi systemd[1]: Started Cleanup of Temporary Directories.
Oct  7 17:17:01 raspberrypi CRON[30021]: (root) CMD (   cd / && run-parts --report /etc/cron.hourly)
Oct  7 18:17:01 raspberrypi CRON[31339]: (root) CMD (   cd / && run-parts --report /etc/cron.hourly)
Oct  7 19:02:39 raspberrypi systemd[1]: Started Session c2 of user pi.
Oct  7 19:04:52 raspberrypi dbus-daemon[796]: Activating service name='com.teamviewer.TeamViewer.Desktop'
Oct  7 19:04:52 raspberrypi dbus-daemon[796]: Successfully activated service 'com.teamviewer.TeamViewer.Desktop'
Oct  7 19:06:24 raspberrypi dbus-daemon[796]: Activating service name='com.teamviewer.TeamViewer.Desktop'
Oct  7 19:06:24 raspberrypi dbus-daemon[796]: Successfully activated service 'com.teamviewer.TeamViewer.Desktop'
Oct  7 19:10:56 raspberrypi systemd[1]: Started Session c3 of user pi.
Oct  7 19:10:59 raspberrypi systemd[1]: Stopping LSB: Start daemon at boot time...
Oct  7 19:10:59 raspberrypi homebridge[32734]: Not running
Oct  7 19:10:59 raspberrypi systemd[1]: Stopped LSB: Start daemon at boot time.
Oct  7 19:10:59 raspberrypi systemd[1]: Starting LSB: Start daemon at boot time...
Oct  7 19:10:59 raspberrypi homebridge[32742]: Starting homebridge
Oct  7 19:10:59 raspberrypi systemd[1]: Started LSB: Start daemon at boot time.

Thx for help
Stefan

**Comment by vplustute** [comment]
![img_2087](https://user-images.githubusercontent.com/34341578/46743515-8bb16c00-cca9-11e8-81a6-8aa88c981b4f.PNG)

Today I had the Problem again.

**Comment by vplustute** [comment]
[homebridge.log](https://github.com/nfarina/homebridge/files/2465114/homebridge.log)

Here is also the hombridge.log

**Comment by gonzaloparra** [comment]
Hi,
Could you fix it? I have the same problem.

**Comment by vplustute** [comment]
>           Hi,
> Could you fix it? I have the same problem.

No, I don`t know what the problem is.
I'm not an expert. That is the reason why I asking for help.
Do you find somthing out?

**Comment by vplustute** [comment]
Can nobody help us?

**Comment by KhaosT** [comment]
@vplustute can you share which router are you using right now? Make sure the router you use does not filter the mDNS packets.

**Comment by vplustute** [comment]
So, what can I do next?

**Comment by vplustute** [comment]
At my Fritzbox Setting I found something like DNS-Rebind-Schutz. I don't know what it is but can I fix here something?

Shout I try to use the pi-hole to fix the Problem? What does pi-hole do?

**Comment by hellspawn0815** [comment]
> At my Fritzbox Setting I found something like DNS-Rebind-Schutz. I don't know what it is but can I fix here something?
https://avm.de/service/fritzbox/fritzbox-7390/wissensdatenbank/publication/show/663_DNS-Aufloesung-privater-IP-Adressen-nicht-moeglich/
https://en.wikipedia.org/wiki/DNS_rebinding

> Shout I try to use the pi-hole to fix the Problem? What does pi-hole do?
Pi-Hole filters DNS request certain ad-driven Websites, so ads can't be displayed if you are surfing www.spiegel.de . If you haven't installed it yet, I think it wouldn't fix your problem.

But the DNS-Rebind-Schutz could be a possible solution for my homebridge+pi-hole problem. Maybe it should allow .local or something like that https://jonathanmumm.com/tech-it/mdns-bonjour-bible-common-service-strings-for-various-vendors/

**Comment by vplustute** [comment]
Sorry I misunderstood, you install pi-hole. 

I'm no expert so I have to read all the things what you gave me and have to check out what I have to type in, at the empty box for DNS-Rebind-Schutz

**Comment by weissertigerxl** [comment]
Did you solve your problem? I have similar problem, but not after some days it is more hours. The plain does not react. Seems other plugin still work. :-( Rebooting heimbringe and the plugin react again. Do You have any idea?

**Comment by vplustute** [comment]
I have absolutely no idea. I think somehow there is still no solution to fix my problem.
I like the homebridge but at this way I can't use it. At the moment I disabled the raspi. I will  set up a new homebridge at the raspi where my loxberry is running. Maybe a new Installation will fix the problem

**Comment by hellspawn0815** [comment]
For me I solved my problem. I have pi-hole installed von my raspberry pi. In my faulty configuration setup I entered the pi-hole IP in my FritzBox router. So EVERY DNS request was forwared to the pi-hole. And I am afraid also raspi DNS requests where also forwared to the pi-hole, so to itself. 

So I changed it at follows: My router has again normal internet DNS Server configured. On my clients, which I use for surfing, I manually added the address of the pi-hole. Now everything works fine 

PS: I also added a command on the raspy on the startup to disable power management for the wifi
